projects/storefrontstyles/scss/layout/page-templates/_category-page.scss

```diff
@@ -19,4 +19,16 @@
       }
     }
   }
+
+  // cx-page-slot {
```

Comment:
I've just pushed a change for this, which moves the focus related css rules to the banner component. that's where it belongs in my pov.